agents/timestream_aggregator/timestream_aggregator.py

```diff
@@ -243,6 +248,18 @@ def read_frames(self):
 
         self.frames = self.reader.Process(None)
         if self.frames:
+            # Handle flow control frames
+            for f in self.frames:
+                if f.type in [core.G3FrameType.none]:
+                    # START; create_new_file
+                    if f.get('sostream_flowcontrol') == 1:
```

Comment:
Please define constants for these flow control values (0,1,2,3), and use those instead of literals.  In the simulator, too.

Reply:
Mhm, great suggestion! Defined an Enum for it in both.